core/templates/dev/head/services/contextual/UrlService.ts

```diff
@@ -138,6 +147,24 @@ oppia.factory('UrlService', ['$window', function($window) {
         return decodeURIComponent(pathname.split('/')[2]);
       }
       throw Error('Invalid collection URL');
+    },
+    getCollectionIdFromEditorUrl: function() {
+      var pathname = this.getPathname();
+      if (pathname.match(/\/(collection_editor\/create)/g)) {
+        return decodeURIComponent(pathname.split('/')[3]);
+      }
+      throw Error('Invalid collection editor URL');
+    },
+    getExplorationVersionFromUrl: function() {
+      if (this.getCurrentQueryString().includes('?v')) {
```

Comment:
Instead of this check if `v` is in `this.getUrlParams()`.

Reply:
Done